src/styles/blocks/nav.scss

```
@@ -0,0 +1,55 @@
+.nav {
+  padding-right: 50px;
+
+  &_list {
+    list-style-type: none;
+    display: flex;
+    align-items: center;
+    text-transform: uppercase;
+    margin: 0;
+    padding: 0;
+  }
+
+  &_item {
+    margin-left: 20px;
+  }
+
+  &_item:first-child {
+    margin-left: 0;
+  }
+
+  &-link,
+  &_item__is-active {
```

it's a modifier, not an element
```suggestion
  &_item--is-active {
```